test/rules/std_AL09_CP02_RF06_combo_test.py

```diff
@@ -0,0 +1,194 @@
+"""Test the interactions of AL09, CP02 & RF06.
+
+AL09: Self aliasing
+CP02: Identifier Capitalisation
+RF06: Identifier Quoting
+"""
+
+import pytest
+
+import sqlfluff
+
+input_query = """
+select
+    a as A,
+    B as b,
+    "C" as C,
+    "d" as d,
+    "E" as e,
+    f as F,
```

Comment:
I believe this is same as `a as A` and can be removed.

Reply:
agreed